tests/flush_database.js

```diff
@@ -0,0 +1,19 @@
+module.exports = () => {
+  const mongoose = require('mongoose');
+
+  mongoose.Promise = global.Promise;
+
+  const dbURI = 'mongodb://localhost/open-platform-dev';
+
+  mongoose.connect(dbURI, {
+    useMongoClient: true
+  });
+
+  mongoose.connection.on('open', () => {
+    mongoose.connection.db.dropDatabase(function (err, res) {
+      console.log(err);
+      console.log(res);
+      process.exit(0);
```

Comment:
If you must have console.logs in here at least make it something like `console.log(err || res)` :stuck_out_tongue_winking_eye: